docs: turn node manager readme into a guide

jacderida opened the pull request:
The README for the node manager was initially generated mostly by ChatGPT, just to flesh it out for first publish. It was based on the UI, so it quickly became out of date.

Some users in the most recent testnet wanted to refer to it to help them with the node manager, so it has now been turned into a guide.

## Description

reviewpad:summary

joshuef reviewed (approved):
Awesome. 

It'll be good to get this looped into CI runs and uploaded. Could be a handy initial indicator of what is off w/  a network

joshuef commented:
bors r+

davidrusu commented:
bors r+

davidrusu commented:
bors r+

jacderida committed:
docs: turn node manager readme into a guide

The README for the node manager was initially generated mostly by ChatGPT, just to flesh it out for
first publish. It was based on the UI, so it quickly became out of date.

Some users in the most recent testnet wanted to refer to it to help them with the node manager, so
it has now been turned into a guide.

joshuef reviewed (approved):
lovely stuff!
